util/opds_writer.py

```diff
@@ -5,6 +5,16 @@
 from lxml import builder, etree
 from nose.tools import set_trace
 
+class ElementMaker(builder.ElementMaker):
```

Comment:
@leonardr helped fixed this pickle issue:
![Screen Shot 2019-07-30 at 5 02 23 PM](https://user-images.githubusercontent.com/1280564/62245109-d3c38b80-b3ae-11e9-98ad-f613693cbb8d.png)